docs/Options.md

```diff
@@ -184,27 +239,38 @@ By default, Battery section is shown only if battery level is below `SPACEFISH_B
 | `true` | Shown | Hidden | Hidden |
 | `charged` | Shown | Hidden | Shown |
 
-### Ruby (`ruby`)
+### Jobs \(`jobs`\)
 
-Ruby section is shown only in directories that contain `Gemfile`, or `Rakefile`, or any other file with `.rb` extension.
+This section show only when there are backgrounded jobs.
```

Comment:
I know this line was carried over from spaceship, but I'd suggest fixing up the English a little. 😅